internal/cmd/server_test.go

```diff
@@ -124,18 +124,12 @@ func TestServerCmd_LoadOptions(t *testing.T) {
 			name: "env vars with config file",
 			setup: func(t *testing.T, cmd *cobra.Command) {
 				content := `
-grants:
-  - user: user1
-    resource: infra
-    role: admin
-  - user: user2
-    resource: infra
-    role: admin
 
 users:
   - name: username
     accessKey: access-key
     password: the-password
+    infraRole: admin
```

Comment:
Fantastic clean up in this PR. It's most likely better to visit this in a follow up PR, but could we further simplify this by removing `users:`? (can move this to a new issue if so)

On first thought, a minimal bootstrap interface could be:

1. A single `accessKey` field for bootstrapping the system via terraform and for follow-on support admin needs
2. A username+password (for bootstrapping via the dashboard) although I don't know if this is a great experience. Quite a few tools would allow just allow the user to sign up as usual (of course would require multi-org by default) after standing up the server by visiting the endpoint it exposes in the browser